src/components/Controls.vue

```diff
@@ -314,11 +383,59 @@ export default {
 			this.$store.dispatch('setFilter', { ...filterReset })
 			this.filter = filterReset
 		},
+		async loadStacksFromBoard(selectedBoard) {
+			try {
+				const url = generateUrl('/apps/deck/stacks/' + selectedBoard.id)
+				const response = await axios.get(url)
+				this.stacksFromBoard = response.data
+			} catch (err) {
+				return err
+			}
+		},
+		async addCard() {
+			try {
+				const today = new Date()
+				today.setHours(23, 59, 59, 999)
```

Comment:
I'm not sure about this, having a card due at 23:59 is probably not the best attempt.

How about adding a date picker and setting a default due date for in an hour there?

Reply:
Most of Todo Tools use Today as default. 
Regarding https://github.com/nextcloud/deck/issues/2231#issuecomment-679945453 Today is the current date and 23:59

Reply:
In #1157 we are discussing the different purpose of a due date at a specific time/timezone and a due date at a day. 

Regarding Desk and the current issues 23:59 seems to be a quick win for me, that can be imoroved later.